sdk/python/test/sdk_unittest/test_training_client.py

```diff
@@ -0,0 +1,193 @@
+import multiprocessing
+import unittest
+from unittest.mock import patch, Mock
+from parameterized import parameterized
+
+from typing import Optional
+from kubeflow.training import TrainingClient
+from kubeflow.training import KubeflowOrgV1ReplicaSpec
+from kubeflow.training import KubeflowOrgV1PyTorchJob
+from kubeflow.training import KubeflowOrgV1PyTorchJobSpec
+from kubeflow.training import KubeflowOrgV1RunPolicy
+from kubeflow.training import KubeflowOrgV1SchedulingPolicy
+from kubeflow.training import constants
+
+from kubernetes.client import V1PodTemplateSpec
+from kubernetes.client import V1ObjectMeta
+from kubernetes.client import V1PodSpec
+from kubernetes.client import V1Container
+from kubernetes.client import V1ResourceRequirements
+
+CONTAINER_NAME = "pytorch"
+JOB_NAME = "pytorchjob-mnist-ci-test"
```

Comment:
Do you need those vars if you use them only in a single place ?

Reply:
I will remove them